.github/pull_request_template.md

```diff
@@ -17,7 +17,7 @@ Changes proposed in this pull request:
 ### Checklist
 #### Implementation
 - [ ] Implementation is correct (it performs its intended function).
-- [ ] Code complies with PEP-8 and is consistent with the framework.
+- [ ] Code is consistent with the framework.
```

Comment:
Could you elaborate on the reasoning of this change?

Reply:
Isn't the same as `CI/CD pipelines run with no errors` since we check the PEP8 style?  
If it's not that clear, I'll revert the change.

Reply:
I think I see your point. So, this specific check is more related to naming conventions and other standards adopted in skmultiflow, whereas linting would be covered in the `CI/CD` portion. Is that right?

Reply:
Absolutely!

Reply:
Perfect!